scripts/outer/kubectl-kubestellar-prep_for_syncer

```diff
@@ -102,6 +110,10 @@ fi
 
 set -e
 
+imw_space_config="${PWD}/temp-space-config/spaceprovider-default-${imw}"
+kubectl-kubestellar-get-config-for-space --space-name ${imw} --provider-name default --sm-core-config $SM_CONFIG --sm-context $SM_CONTEXT --space-config-file $imw_space_config
```

Comment:
A similar comment to https://github.com/kubestellar/kubestellar/pull/1268/files#r1411549630 applies here. For `kubectl-kubestellar-prep_for_syncer` I think that perhaps the best approach is that the script is given command line flags and `$KUBECONFIG` that point at the space that holds the space management API objects and has optional parameters for the name of the IS (IMW) and KCS (ESPW).